packaging/systemd/miniflux.service

```diff
@@ -3,61 +3,94 @@
 # All your changes can be reverted with `systemctl revert miniflux.service`.
 # See https://wiki.archlinux.org/index.php/Systemd#Editing_provided_units.
+# Also see https://www.freedesktop.org/software/systemd/man/systemd.service.html
+# for available configuration options in this file.
 
 [Unit]
 Description=Miniflux
 After=network.target postgresql.service
 
 [Service]
 ExecStart=/usr/bin/miniflux
-EnvironmentFile=/etc/miniflux.conf
 User=miniflux
 
-# https://www.freedesktop.org/software/systemd/man/systemd.service.html#Type=
+# Load environment variables from /etc/miniflux.conf.
+EnvironmentFile=/etc/miniflux.conf
+
+# Miniflux uses sd-notify protocol to notify about it's readiness.
 Type=notify
 
-# https://www.freedesktop.org/software/systemd/man/systemd.service.html#WatchdogSec=
+# Enable watchdog.
 WatchdogSec=60s
 WatchdogSignal=SIGKILL
 
-# https://www.freedesktop.org/software/systemd/man/systemd.service.html#Restart=
+# Automatically restart Miniflux if it crashes.
 Restart=always
-
-# https://www.freedesktop.org/software/systemd/man/systemd.service.html#RestartSec=
 RestartSec=5
 
-# https://www.freedesktop.org/software/systemd/man/systemd.exec.html#NoNewPrivileges=
-NoNewPrivileges=true
+# Allocate a directory at /run/miniflux for Unix sockets.
+RuntimeDirectory=miniflux
 
-# https://www.freedesktop.org/software/systemd/man/systemd.exec.html#PrivateDevices=
-PrivateDevices=true
+# Allow Miniflux to bind to privileged ports.
+AmbientCapabilities=CAP_NET_BIND_SERVICE
 
-# https://www.freedesktop.org/software/systemd/man/systemd.exec.html#ProtectControlGroups=
-ProtectControlGroups=true
+# Make the system tree read-only.
+ProtectSystem=strict
 
-# https://www.freedesktop.org/software/systemd/man/systemd.exec.html#ProtectHome=
-ProtectHome=true
+# Allocate a separate /tmp.
+PrivateTmp=yes
 
-# https://www.freedesktop.org/software/systemd/man/systemd.exec.html#ProtectKernelModules=
-ProtectKernelModules=true
+# Ensure the service can never gain new privileges.
+NoNewPrivileges=yes
 
-# https://www.freedesktop.org/software/systemd/man/systemd.exec.html#ProtectKernelTunables=
-ProtectKernelTunables=true
+# Prohibit access to any kind of namespacing.
+RestrictNamespaces=yes
 
-# https://www.freedesktop.org/software/systemd/man/systemd.exec.html#ProtectSystem=
-ProtectSystem=strict
+# Make home directories inaccessible.
+ProtectHome=yes
 
-# https://www.freedesktop.org/software/systemd/man/systemd.exec.html#RestrictRealtime=
-RestrictRealtime=true
+# Make device nodes except for /dev/null, /dev/zero, /dev/full,
+# /dev/random and /dev/urandom inaccessible.
+PrivateDevices=yes
 
-# Keep at least the /run folder writeable if Miniflux is configured to use a Unix socket.
-# For example, the socket could be LISTEN_ADDR=/run/miniflux/miniflux.sock
-# https://www.freedesktop.org/software/systemd/man/systemd.exec.html#ReadWritePaths=
-ReadWritePaths=/run
+# Make users other than root and the user for this daemon
+# inaccessible.
+PrivateUsers=yes
```

Comment:
This option prevents Miniflux to start if `LISTEN_ADDR=:http`. It crashes with the following error even with `AmbientCapabilities=CAP_NET_BIND_SERVICE`:

```
Dec 16 04:38:38 srv01.miniflux.net miniflux[23457]: [INFO] Listening on ":http" without TLS
Dec 16 04:38:38 srv01.miniflux.net miniflux[23457]: [FATAL] Server failed to start: listen tcp :80: bind: permission denied
Dec 16 04:38:38 srv01.miniflux.net systemd[1]: miniflux.service: Main process exited, code=exited, status=1/FAILURE
Dec 16 04:38:38 srv01.miniflux.net systemd[1]: miniflux.service: Failed with result 'exit-code'.
```

Reply:
Removed it.